Snabble/UI/DynamicStack/WidgetTextView.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  WidgetTextView.swift
+//  Snabble
+//
+//  Created by Uwe Tilemann on 01.09.22.
+//
+
+import SwiftUI
+
+public enum ColorStyle: String {
+    case label
+    case secondaryLabel
+    case accent
+    case onAccent
+    case border
+    case shadow
+    
+    var color: SwiftUI.Color {
+        switch self {
+        case .label:
+            return Color.label
+        case .secondaryLabel:
+            return Color.secondaryLabel
+        case .accent:
+            return Color.accent()
+        case .onAccent:
+            return Color.onAccent()
+        case .border:
+            return Color.border()
+        case .shadow:
+            return Color.shadow()
+            
+        default:
+            break
+        }
+        print("getting color for \(self.rawValue)")
+        return Color(self.rawValue)
+    }
+}
+
+public enum TextStyle: String {
+    case body
+    case footnote
+    case headline
+    case title
+}
```

Comment:
Was hältst du davon, wie mein Vorschlag bei Color, das ganze in den Snabble/UI als Utility auszulagern?